templates/workspaces/base/terraform/keyvault.tf

```diff
@@ -3,7 +3,7 @@ resource "azurerm_key_vault" "kv" {
   location                 = azurerm_resource_group.ws.location
   resource_group_name      = azurerm_resource_group.ws.name
   sku_name                 = "standard"
-  purge_protection_enabled = var.keyvault_purge_protection_enabled
+  purge_protection_enabled = true
```

Comment:
are we removing the ability to turn purge protection off?

Reply:
I assume we are...just wondered if we should continue to pass this as a variable, but default to `true`

Reply:
With the latest provider - I think this should always be true and we soft delete and recover secrets at the provider level. There are too many cases where the dev cycles is different to the CI or Prod flow.

Reply:
fair enough, but what is `KEYVAULT_PURGE_PROTECTION_ENABLED` in the sample env file? Should we remove that too?

Reply:
Removed all capabilities to disable purge protection